mne/viz/tests/test_3d.py

```diff
@@ -103,6 +105,19 @@ def test_plot_sparse_source_estimates():
     plot_sparse_source_estimates(sample_src, stc, bgcolor=(1, 1, 1),
                                  opacity=0.5, high_resolution=False)
 
+    # Test plotting with mpl.
+    stc.plot(subjects_dir=subjects_dir, time_unit='s', views='ven', hemi='rh',
+             smoothing_steps=2, subject='sample', backend='matplotlib',
+             spacing='all', initial_time=0.001, colormap='Reds')
```

Comment:
On my computer the mpl tests take around 14 seconds (mostly on the ``spacing=all`` case). I'm not sure if it's possible to speed up the ``all`` case, but I think we should keep the test.